src/test/ui/lint-unconditional-recursion.stderr

```diff
@@ -1,157 +1,157 @@
 
 error: function cannot return without recurring
   --> $DIR/lint-unconditional-recursion.rs:96:5
    |
-96 |     fn as_ref(&self) -> &Self { //~ ERROR function cannot return without recurring
+LL |     fn as_ref(&self) -> &Self { //~ ERROR function cannot return without recurring
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^ cannot return without recurring
-97 |         Baz::as_ref(self)
+LL |         Baz::as_ref(self)
    |         ----------------- recursive call site
    |
    = help: a `loop` may express intention better if this is on purpose
 
 error: function cannot return without recurring
-   --> $DIR/lint-unconditional-recursion.rs:103:5
-    |
-103 |     fn default() -> Baz { //~ ERROR function cannot return without recurring
-    |     ^^^^^^^^^^^^^^^^^^^ cannot return without recurring
-104 |         let x = Default::default();
-    |                 ------------------ recursive call site
-    |
-    = help: a `loop` may express intention better if this is on purpose
+  --> $DIR/lint-unconditional-recursion.rs:103:5
+   |
+LL |     fn default() -> Baz { //~ ERROR function cannot return without recurring
+   |     ^^^^^^^^^^^^^^^^^^^ cannot return without recurring
+LL |         let x = Default::default();
+   |                 ------------------ recursive call site
+   |
+   = help: a `loop` may express intention better if this is on purpose
 
 error: function cannot return without recurring
-   --> $DIR/lint-unconditional-recursion.rs:112:5
-    |
-112 |     fn deref(&self) -> &() { //~ ERROR function cannot return without recurring
-    |     ^^^^^^^^^^^^^^^^^^^^^^ cannot return without recurring
-113 |         &**self
-    |          ------ recursive call site
-    |
-    = help: a `loop` may express intention better if this is on purpose
+  --> $DIR/lint-unconditional-recursion.rs:112:5
+   |
+LL |     fn deref(&self) -> &() { //~ ERROR function cannot return without recurring
+   |     ^^^^^^^^^^^^^^^^^^^^^^ cannot return without recurring
+LL |         &**self
+   |          ------ recursive call site
+   |
+   = help: a `loop` may express intention better if this is on purpose
 
 error: function cannot return without recurring
-   --> $DIR/lint-unconditional-recursion.rs:119:5
-    |
-119 |     fn index(&self, x: usize) -> &Baz { //~ ERROR function cannot return without recurring
-    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot return without recurring
-120 |         &self[x]
-    |          ------- recursive call site
-    |
-    = help: a `loop` may express intention better if this is on purpose
+  --> $DIR/lint-unconditional-recursion.rs:119:5
+   |
+LL |     fn index(&self, x: usize) -> &Baz { //~ ERROR function cannot return without recurring
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot return without recurring
+LL |         &self[x]
+   |          ------- recursive call site
+   |
+   = help: a `loop` may express intention better if this is on purpose
```

Comment:
Example of a test with "long" actual numbers that become "short" anonymized numbers.